docs/source/specs/openapi.json

```diff
@@ -1305,6 +1331,24 @@
             "schema": {
               "type": "string"
             }
+          },
```

Comment:
Do we want these to be added now? That would generate clients with this capability, which doesn't actually exist (yet).